src/core/backbone_router/bbr_local.cpp

```diff
@@ -267,7 +267,12 @@ void Local::HandleBackboneRouterPrimaryUpdate(Leader::State aState, const Backbo
     {
         // Here original PBBR restores its Backbone Router Service from Thread Network,
         // Intentionally skips the state update as PBBR will refresh its service.
-        mSequenceNumber      = aConfig.mSequenceNumber + 1;
+#if OPENTHREAD_CONFIG_REFERENCE_DEVICE_ENABLE
+        // BBR-TC-02 forces Sequence Number for the reference device with raw UDP API
+        mSequenceNumber = aConfig.mSequenceNumber;
```

Comment:
I had one concern whether or not there is possible impact on DUT's spontaneous DUA/MLR re-registration check after reference PBBR reboot. After quick glance over the test plan, there seems no such test cases and it's always the test harness instructs to increase sequence number. I am fine with this proposal. thanks.